create_magic.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ARCH=x86_64
+TYPE=qcow2
+DIR=~/build
+mkdir $DIR
+DIST=fedora21
+DOMAIN=$DIST-cockpit
```

Comment:
this is for project cockpit, I have to create some specialized instances. So this create_magic.sh is more-less only example how to create virt-builder repo automatically. It is more less template script.